css/main.css

```diff
@@ -560,3 +561,58 @@ textarea {
     height: 32px;
   }
 }
+
+.toggle-menu {
+  display: block;
+  height: 100vh;
+  width: 100vw;
+  background: rgb(60 58 57 / 90%);
+  z-index: 10;
+  padding: 62px 28px;
+}
+
+.remove-padding {
+  padding: 0;
+}
+
+.toggle-nav .nav-list {
+  display: flex;
+  flex-direction: column;
+  padding-top: 32px;
+}
+
+.toggle-nav .nav-list .nav-item {
+  border-bottom: 2px solid #6f6c6b;
+}
+
+.toggle-nav .nav-list .nav-item a {
+  font-size: 32px;
+  font-weight: 600;
+  color: #fff5e1;
+  margin-bottom: 40px;
+}
```

Comment:
Good job following the design on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) 😁 but it still needs some touches to be perfect, kindly try to match it with the figma design as close as you can, currently your design is like this:
<img src="https://user-images.githubusercontent.com/20567503/175565509-1cc326f3-7d8e-483a-aae1-bf32ecc92d06.png" width="300">

Whereas the Figma design looks like this:

<img src="https://user-images.githubusercontent.com/20567503/175565589-55027404-d0b1-4550-a2cb-7a846787aafb.png" width="300">

Please know that matching the design would make your portfolio look much better and more professional 😎😎.